src/Files.App/ViewModels/Properties/HashesViewModel.cs

```diff
@@ -23,11 +26,47 @@ public HashInfoItem SelectedItem
 		public Dictionary<string, bool> ShowHashes { get; private set; }
 
 		public ICommand ToggleIsEnabledCommand { get; private set; }
+		public ICommand CompareFileCommand { get; private set; }
 
 		private ListedItem _item;
 
 		private CancellationTokenSource _cancellationTokenSource;
 
+		private string _hashInput;
+		public string HashInput
+		{
+			get => _hashInput;
+			set
+			{
+				SetProperty(ref _hashInput, value);
+
+				OnHashInputTextChanged();
+				OnPropertyChanged(nameof(IsInfoBarOpen));
+				IsInfoBarOpen = !string.IsNullOrEmpty(value);
```

Comment:
```suggestion
				OnHashInputTextChanged();
				OnPropertyChanged(nameof(IsInfoBarOpen));
```

Reply:
`OnPropertyChanged(nameof(IsInfoBarOpen));` takes care of updating the property so we don't need to manually assign it.